yt/frontends/ramses/io.py

```diff
@@ -214,7 +214,9 @@ def _read_part_file_descriptor(fname):
     Read a file descriptor and returns the array of the fields found.
     """
     VERSION_RE = re.compile(r'# version: *(\d+)')
-    VAR_DESC_RE = re.compile(r'\s*(\d+),\s*(\w+),\s*(\w+)')
+    # This will match any line that look like
+    #    01, foo_bar, baz
+    VAR_DESC_RE = re.compile(r'\s*([^,]+),\s*([^,]+),\s*([^,]+)')
```

Comment:
@suniverse points out there's another regex on line 276 below that also needs to be updated.

Reply:
That's right. I've factorized the two regexp as they are the same and moved to `definitions.py`